src/schema/v2/homeView/HomeViewComponent.ts

```diff
@@ -0,0 +1,31 @@
+import { GraphQLEnumType, GraphQLObjectType } from "graphql"
+
+// known standard component types
+
+const HomeViewComponentTypeType = new GraphQLEnumType({
+  name: "HomeViewComponentType",
+  description: "Known component types for use in home view",
+  values: {
+    ARTWORKS_RAIL: {
+      value: "artworks_rail",
```

Comment:
We should avoid snake case

Reply:
Fixed in 397eb2849

Reply:
Agreed its all over the place, and fixing stuff like this en-masse is delicate... Just thinking of JS norms, and small nudges forward in that way (similar to how we've evolved to name files in `camelCaseFormat.ts`.